make/compiler/Makefile.gnu

```diff
@@ -267,6 +267,13 @@ ifeq ($(shell test $(GNU_GPP_MAJOR_VERSION) -eq 9; echo "$$?"),0)
 WARN_CXXFLAGS += -Wno-error=init-list-lifetime
 endif
 
+#
+# Avoid false positive warnings about use-after free with realloc
+# that occur in GCC 12.
+#
+ifeq ($(shell test $(GNU_GPP_MAJOR_VERSION) -eq 12; echo "$$?"),0)
```

Comment:
`-ge 12` or are we waiting until 13 is out to see if the bug is still present before making that change?

Reply:
I would like to find out when 13 comes out if the bug is still present, yes